library/account/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,3 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+</manifest>
```

Comment:
This may not be necessary now since you are using the `StateFlow` provided in `:library:base`